packages/sysutils/busybox/scripts/init

```diff
@@ -34,6 +34,9 @@
   /bin/busybox mount -t proc proc /proc
   /bin/busybox mount -t sysfs sysfs /sys
 
+  # Just in case... problems with ssh if not correctly owned
+  /bin/busybox chown 0:0 /storage 2>/dev/null
```

Comment:
sorry. think I didnt express myself right in my previous comment. /bin/busybox not needed here once /proc is mounted, but does not harm of course. not bikeshedding ;)